src/OSPSuite.Core/Serialization/Xml/OptimizationRunPropertiesXmlSerializer.cs

```diff
@@ -7,6 +7,7 @@ public class OptimizationRunPropertiesXmlSerializer : OSPSuiteXmlSerializer<Opti
       public override void PerformMapping()
       {
          Map(x => x.NumberOfEvaluations);
+         Map(x => x.DateTimeCompleted);
```

Comment:
Means this too is not compatible with V12

Reply:
potentially yes. I am not sure about attributes